CONTRIBUTING.md

```diff
@@ -66,3 +66,26 @@ GIT_USER=<Your GitHub username> USE_SSH=true yarn deploy
 ```
 
 If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
+
+### Search
+
+To build index for search
+
+```text
+1. you would need to sign up an account on https://www.algolia.com/users/sign_in
+2. create a project for the purpose of creating index
+3. add an INDEX_NAME called "docusaurus" for instance on same website
+4. You add API_key and give rights for ACLs search, addObject, deleteObject, editSearch for the key
```

Comment:
Wrap all the technical bits of this in backticks.  Also, confirm that it's API_key vs. API_KEY.